driver/src/main/java/org/neo4j/driver/internal/async/pool/NettyChannelTracker.java

```diff
@@ -41,11 +45,18 @@
     private final Logger log;
     private final MetricsListener metricsListener;
     private final ChannelFutureListener closeListener = future -> channelClosed( future.channel() );
+    private final ChannelGroup allChannels;
 
-    public NettyChannelTracker( MetricsListener metricsListener, Logging logging )
+    public NettyChannelTracker( MetricsListener metricsListener, EventExecutorGroup eventExecutorGroup, Logging logging )
```

Comment:
I think this can just take and `EventExecutor` and pass it directly to the `DefaultChannelGroup` constructor. `bootstrap.config().group().next()` returns an `EventExecutor`.